gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/source/extractor/extract/kafka/workunit/packer/KafkaTopicGroupingWorkUnitPacker.java

```diff
@@ -187,6 +188,10 @@ public KafkaTopicGroupingWorkUnitPacker(AbstractSource<?, ?> source, SourceState
         containerCapacity = getContainerCapacityForTopic(capacitiesByTopic.get(topic), this.containerCapacityComputationStrategy);
         log.info("Container capacity for topic {}: {}", topic, containerCapacity);
       }
+      //Add CONTAINER_CAPACITY into each workunit. Useful when KafkaIngestionHealthCheck is enabled.
```

Comment:
Do you want call addContainerCapacityToWorkunits() here?

Reply:
Forgot to remove the method after refactoring. Fixed it.